code/modules/mob/living/carbon/give.dm

```diff
@@ -0,0 +1,56 @@
+/mob/living/carbon/verb/give(var/mob/living/carbon/target in oview(1))
+	set category = "IC"
+	set name = "Give"
+
+	if(target.stat == UNCONSCIOUS)
+		to_chat(usr, "<span class='warning'>[target.name] is unconscious!</span>")
+		return
+	else if(target.stat == DEAD)
+		to_chat(usr, "<span class='warning'>[target.name] is dead!</span>")
+		return
+	else if(target.stat != CONSCIOUS)
+		to_chat(usr, "<span class='warning'>[target.name] cannot receive your item!</span>")
+		return
+
+	if(!iscarbon(target)) //something is bypassing the give arguments, no clue what, adding a sanity check JIC
+		to_chat(usr, "<span class='danger'>Wait a second... \the [target] HAS NO HANDS! AHH!</span>")//cheesy messages ftw
+
+		return
+	if(target.stat == 2 || usr.stat == 2|| target.client == null)
```

Comment:
Don't you essentially do this in lines 5-13?

Also lets not magic number stat... lets use the defines.